src/components/app.js

```diff
@@ -144,6 +150,54 @@ const App = () => {
         }
     }, [user]);
 
+    const connectNotificationsUpdateConfig = useCallback(() => {
+        const ws = connectNotificationsWsUpdateConfig();
+
+        ws.onmessage = function (event) {
+            fetchConfigParameters().then((params) => {
+                params.forEach((param) => {
```

Comment:
It is necessary to get all parameters ?
do not need a fetch for a single parameter ?